lib/helpers/measure_helpers.js

```diff
@@ -355,22 +343,10 @@ module.exports = class MeasureHelpers {
      * @param {string} statementName - The statement name to search for.
      * @return {boolean} If the statement is a function or not.
      */
-  static isStatementFunction(measure, libraryName, statementName) {
-    // find the library and statement in the elm.
-    let library = null;
-    let statement = null;
-    for (const lib of Array.from(measure.elm)) {
-      if (lib.library.identifier.id === libraryName) {
-        library = lib;
-      }
-    }
-    for (const curStatement of Array.from(library.library.statements.def)) {
-      if (curStatement.name === statementName) {
-        statement = curStatement;
-      }
-    }
-
-    if (library != null && statement != null) {
+  static isStatementFunction(library, statementName) {
+    // find the library and statement in the elm
+    const statement = library.elm.library.statements.def.find(def => def.name === statementName);
+    if (statement != null) {
```

Comment:
When `find` does not return results, it returns `undefined`. Does that mean this should check for `statement != undefined`?

Reply:
![image](https://user-images.githubusercontent.com/3967953/52065571-cbe07700-2544-11e9-90e0-9b96d51e079e.png)

Reply:
Should be fine